src/angular/src/app/pages/files/file-list.component.html

```
@@ -1,41 +1,41 @@
-                (extractEvent)="onExtract($event)"
-                (validateEvent)="onValidate($event)"
-                (deleteLocalEvent)="onDeleteLocal($event)"
-                (deleteRemoteEvent)="onDeleteRemote($event)">
-            </app-file>
-        </div>
-    }
-</div>
+<div id="file-list">
+    <div id="header">
+        <div class="checkbox">
+            <input type="checkbox" aria-label="Select all files" (change)="$any($event.target).checked ? onCheckAll() : onUncheckAll()" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)